src/v2/Apps/Partner/Components/ScrollToPartnerHeader/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { ScrollIntoView } from "v2/Utils"
```

Comment:
Yikes, we need to get rid of this component. It's a terrible pattern. Can you deprecate it and then just use `scrollIntoView` directly on the appropriate buttons?

Reply:
As to why it's a terrible pattern: it makes no sense to just bind a click handler to a random outer div when you have the function itself that you can include on the correct target. It's not focusable via keyboard which makes this inaccessible: you tab into something and "click" it and this won't scroll the target into view.

Reply:
I use Media here to set the right offset. I can use `useMatchMedia` hook, is it okay here?

Reply:
Updated `ScrollToPartnerHeader` component, please take a look. now it works via keyboard.

![ezgif com-gif-maker (20)](https://user-images.githubusercontent.com/79979820/117040982-231cb580-ad13-11eb-8d04-64303ba38412.gif)